app/templates/components/public/ticket-list.hbs

```diff
@@ -1,11 +1,11 @@
 {{!-- Removed `Currency Symbol` helper from the `paymentCurrency` since some currency are not getting rendered. Hotfix until a permanent solution found --}}
 <form class="ui form" {{action on='submit' preventDefault=true}}>
-  <table class="ui unstackable very basic compact table" style="margin-bottom: 0">
+  <table class="ui unstackable {{if device.isMobile 'small'}} very basic compact table" style="margin-bottom: 0">
```

Comment:
<img width="126" alt="Screenshot 2019-10-30 at 7 25 53 PM" src="https://user-images.githubusercontent.com/44091822/67864131-26961600-fb4b-11e9-9632-258f9f82d8bf.png">

Solve CSS Overflow
--

Reply:
sir it due to "ui segment"  class and it is also exist before my implementation in actual website.
![Screenshot from 2019-10-30 20-16-02](https://user-images.githubusercontent.com/56407566/67868903-9fe53700-fb52-11e9-8665-a878cc59c95d.png)
It is visible fine in galaxy5, and pixel2 devices, but showing this overflow in very small devices .

Reply:
please help me with this issue if there is any thing else

Reply:
I will pull and check on local tonight.

Reply:
ok